pkg/network/extpool/pool.go

```diff
@@ -40,6 +47,10 @@ func (p *Pool) Add(e *payload.Extensible) (bool, error) {
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
+	if len(p.verified) >= p.capacity {
+		return false, nil
```

Comment:
Wrong. You need to kick the oldest entry out of the pool. Which BTW is what we had with Neo 2 consensus messages pool implementation (pool.HashCache).

Reply:
Oh, forgot about that, fixed.

Reply:
And I am not sure if we should evict items from the same sender. For DoS mitigation current implementation (don't add) is better IMO.